<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetWidget</class>
 <widget class="QWidget" name="DatasetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="excelButton">
       <property name="text">
        <string>Load from Excel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tsvButton">
       <property name="text">
        <string>Load from Tsv</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="DatasetView" name="datasetView"/>
     <widget class="QTextEdit" name="descriptionDocument"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatasetView</class>
   <extends>QTableView</extends>
   <header>datasetWidget/datasetView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
